doc/nrf/protocols/thread/tools.rst

```diff
@@ -281,13 +281,13 @@ Build the OpenThread POSIX applications by performing the following steps:
       ./script/bootstrap
 
 #. Build the applications with the required options.
-   For example, to build the ``ot-cli`` application with support for Thread v1.1, run the following command::
+   For example, to build the ``ot-cli`` application with support for Thread v1.2, run the following command::
```

Comment:
this part sounds a bit confusing to me. Maybe we could split this into 3 cases explaining default build, changing versions and enabling ot-daemon?
- `./script/cmake-build posix`
- `./script/cmake-build posix -DOT_THREAD_VERSION=1.2`
- `./script/cmake-build posix -DOT_DAEMON=ON`

Reply:
Let's remove info about building other versions of thread since we tend to support only the newest default version. for this we also generate libraries.